funds-manager/funds-manager-server/src/handlers.rs

```diff
@@ -3,13 +3,17 @@
 use crate::custody_client::DepositWithdrawSource;
 use crate::error::ApiError;
 use crate::Server;
-use funds_manager_api::{DepositAddressResponse, WithdrawFundsRequest};
+use funds_manager_api::{DepositAddressResponse, WithdrawFundsRequest, WithdrawGasRequest};
 use std::collections::HashMap;
 use std::sync::Arc;
 use warp::reply::Json;
 
 /// The "mint" query param
 pub const MINT_QUERY_PARAM: &str = "mint";
+/// The asset used for gas (ETH)
+pub const GAS_ASSET_NAME: &str = "ETH";
+/// The maximum amount of gas that can be withdrawn at a given time
+pub const MAX_GAS_WITHDRAWAL_AMOUNT: f64 = 0.1; // 0.1 ETH
```

Comment:
don't mind having this as a constant here, but: 1) we should also make sure this limit is set in FB, and 2) this seems to be unused